### PR TITLE
Create Data Model

### PR DESCRIPTION
**Pull request by kendoriddy**
In this milestone I:

⊛ Install Ruby on Rails framework.
⊛ Use Ruby Gems as a software package system.
⊛ Understand what ORM is.
⊛ Use database migration files to maintain database schema.
⊛ Install and check linters.
⊛ Add a descriptive README file with the new version 2.0.

### CODE REVIEW

File: db/migrate/20221122150342_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
```
Comment (DammyShittu) [inline db/migrate/20221122150342_create_posts.rb]: - [x] The references should point to an `author`, not `user`. You can use the `to_table` to point the database to the User table.

  You can refer to this thread on naming references to point to a different table: [https://stackoverflow.com/questions/13694654/specifying-column-name-in-a-references-migration](https://stackoverflow.com/questions/13694654/specifying-column-name-in-a-references-migration)

File: db/migrate/20221122151644_create_comments.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :user, null: false, foreign_key: true
```
Comment (DammyShittu) [inline db/migrate/20221122151644_create_comments.rb]: - [x] The references should point to an `author`, not `user`. You can use the `to_table` option to point the database to the User table.

File: db/migrate/20221122151902_create_likes.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```
Comment (DammyShittu) [inline db/migrate/20221122151902_create_likes.rb]: - [x] The references should point to an `author`, not `user`. You can use the `to_table` option to point the database to the User table.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @kendoriddy,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] You should use `postgresql` for your database, not `sqlite3`. That means you should have something like this in the `default` section

  ```
  default: &default
    adapter: postgresql
    encoding: unicode
  ```

  Ensure you rename your databases in the development, test and production environments

  image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20221122150342_create_posts.rb (hunk 1 above)
Comment (kendoriddy) [reply db/migrate/20221122150342_create_posts.rb]: @DammyShittu thank you very much for your review and the link. Happy codinng! 🚀

**Review by mikethreels — APPROVED** [review]
Hi Kenny, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20221122151902_create_likes.rb (hunk 3 above)
Comment (kendoriddy) [reply db/migrate/20221122151902_create_likes.rb]: Thank you for the review @DammyShittu

File: db/migrate/20221122151644_create_comments.rb (hunk 2 above)
Comment (kendoriddy) [reply db/migrate/20221122151644_create_comments.rb]: Thank you for the review @DammyShittu

**Comment by kendoriddy** [comment]
Thank you for the review @mikethreels

**Pull request merged**